### ISSUE TITLE
Utilisation de rtab map pour la naviguation

### ISSUE THREAD

**Issue by maximeconnolly** [WalkingMachine/sara_commun#37]
Dans le code, il y a de la reference a rtabmap qui est un algorithme de slam. Il a l'aire a mieux fonctionner dans une environnement plus ouvert sans reference et créer pour des camera kinect et kinect clone. 

http://wiki.ros.org/rtabmap 

Les alternative sont hector slam et gmapping (actuellement utilisé)

**Comment by gortium** [comment]
Yep j'avais fait plusieurs test avec rtabmap et ça semblais prometteur mais trop incertain pour la compé.. Ce serrais le bon temps de remettre ce projet sur les rails ! 
En plus ça vient de l’université de sherbrooke, donc facile a contacter si besoin est !

**Comment by maximeconnolly** [comment]
Cool je vais essayer de faire des test avec la kinect et la xtion quand on aura le temps.

**Comment by JeffCousineau** [comment]
ça vient de fucking FRANCOIS MICHAUD !! ENCORE LUI TBK @red1laref
